internal/proxy/proxy.go

```diff
@@ -23,6 +24,18 @@ const (
 	DefaultBufferSizeBytes = 32 * 1024
 	// DefaultFlushInterval TODO
 	DefaultFlushInterval = time.Duration(-1)
+	// DefaultTimeout TODO
```

Comment:
What is left to do here? Simply right the comment?

Reply:
yes :)